packages/deploy-tooling/test/test-input/ui5-deploy.yaml

```diff
@@ -21,3 +21,4 @@ builder:
         target:
           destination: EXAMPLE
           url: https://target.example
+          sersvice: /bc/my/service
```

Comment:
That test shouldn't be working... or do you have a magic string auto-correction that converts `sersvice` to `service` ;)